ajax/libs/milligram/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "milligram",
+  "version": "1.0.3",
+  "description": "A minimalist CSS framework.",
+  "homepage": "http://milligram.github.io",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:milligram/milligram.git"
+  },
+  "license": "MIT",
+  "author": "CJ Patoilo <cjpatoilo@gmail.com>",
+  "keywords": [
+    "boilerplate",
+    "bootstrap",
+    "css",
+    "css3",
+    "design",
+    "front-end",
+    "framework",
+    "html",
+    "html5",
+    "kickstart",
+    "less",
+    "milligram",
+    "responsive",
+    "mobile",
+    "mobile-first",
+    "responsive",
+    "sass",
+    "stylus",
+    "style",
+    "stylesheet"
+  ],
+  "npmName": "milligram",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+      	"milligram.css",
+      	"milligram.min.css",
+      	"milligram.min.css.map"
```

Comment:
Need one more space here, and please use `"**/*"` instead of three entries, thanks.